Homework1: Anastasia Valkevich.py

```diff
@@ -0,0 +1,376 @@
+import csv
+import glob
+import itertools
+import random
+import re
+import sys
+import unittest
+import xml.etree.cElementTree as groupby
+from itertools import groupby
+from time import localtime
+
+# 1 line: Output
+print('Hello, world!')
+
+# 2 lines: Input, assignment
+name = input('What is your name?\n')
+print('Hi, %s.' % name)
+
+# 3 lines: For loop, built-in enumerate function, new style formatting
```

Comment:
It looks much more clear if you task description stands closely to task and 
separated from other tasks with two blank lines: 
```python
   pass  # last line of previous task
    
 
   # next task docstring
   pass  # my next task first line  
```